python/README.md

```diff
@@ -1251,7 +1251,33 @@ deactivate
 ```
 If you want to re-enter the virtualenv just follow the same instructions above about activating a virtualenv. There’s no need to re-create the virtualenv.
 
-##References
+### VirtualenvWrapper
+Even tough is not mandatory, there is a wrapper that sets some defaults about virtualenv use.
+You can install it with pip following this instructions.
+```
+$ pip install virtualenvwrapper
+```
+Then you should add the following lines to your .profile or .bashrc:
+```
+export VIRTUALENVWRAPPER_PYTHON=/usr/local/bin/python3; # This is to avoid specifiying the python version every time.
+export WORKON_HOME=$HOME/.virtualenvs                   # in Linux Python 3 should be the default and this line  
+export PROJECT_HOME=$HOME/Devel                         # should not be necesary anymore.
+source /usr/local/bin/virtualenvwrapper.sh
+```
+This will give you a couple of handful commands for virtualenv management.
+Then you should be able to create the .virtualenvs folder and you should be able to create virtualenvs with:
```

Comment:
I think there is too much "should be able" in this sentence.
Maybe you can said: "Now you can create a new environment running the following command:", or "now you should be able to create a new environment running the following command" or something like that. I think sounds better.